doc/operator-capabilities.md

```diff
@@ -0,0 +1,193 @@
+# Operator Capability Levels
+
+Operators come in different maturity levels in regards to their lifecycle management capabilities for the application or workload they deliver. The capability models aims to provide guidance in terminology to express what features users can expect from an operator.
+
+![operator-capability-level](./images/operator-capability-level.png)
+
+Each capability level is associated with a certain set of management features the Operator offers around the managed workload. Operator that do not manage a workload and/or are delegating to off-clusters orchestration services would remain at Level 1. Capability levels are accumulating, i.e. Level 3 capabilities require all capabilities desired from Level 1 and 2.
+
+
+## Terminology
+
+**Operator** - the custom controller installed on a Kubernetes cluster
+
+**Operand** - the managed workload provided by the Operator as a service
+
+**Custom Resource (CR)** - an instance of the `CustomResourceDefinition` the Operator ships that represents the Operand or an Operation on an Operand
+
+**Managed resources** - the Kubernetes objects or off-cluster services the Operator uses to constitute an Operand
+
```

Comment:
Also, I'd add: 

Primary Resources and Second Resources.

Reply:
Can you provide details on what the difference is?

Reply:
Primary Resources are the principal one reconcile by the controller. (CR from CRD)
The Secondary Resources are  the second one required for the Kind (CRD)

E.g (The CR of the controller is the Primary Resource and the Deployment, Service and etc are the secondary ones.  )
See: 
https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/pkg/controller/memcached/memcached_controller.go#L46-L61

Reply:
Just one nit the CR can be the second resource if it is not the CR of the operator/controller. I do not know how to clarifies it :-(

Reply:
I think this would classify as watched resources. Like with `Deployments` or `Namespaces`, also CRD owned by other Operators can be watched by an Operator without it being the primary owner. I think as this does not relate to any of the best practices outlined here, adding it would only create confusion.